vector-im/element-x-android
# Investigation of duplicate notification.

## Type of change


## Content


- Add more log and tag to be able to understand issue #609
- Avoid rendering (deleted) notification when user dismiss it manually

## Motivation and context


#609

## Screenshots / GIFs



## Tests



- Did not repro #609, but these change should have no effect on the current impl.
- Send message to the DUT to check notification are correctly displayed and dismissed if for instance the user enter the room.
- See that the log are more useful.

## Tested devices

- OS version(s):

## Checklist
